### DISCUSSION TITLE
[2161050] Boti: Byteland Overclocked （波提：字节国度大冒险）

### DISCUSSION THREAD

**Discussion by ZEERDEER** [pjy612/SteamManifestCache#49] [Share]
[[2161050] Boti Byteland Overclocked （波提：字节国度大冒险）.zip](https://github.com/pjy612/SteamManifestCache/files/13691136/2161050.Boti.Byteland.Overclocked.zip)

**Comment by pjy612** [comment]
感谢丰富库存~
